docs/Developers/Writing_Good_Commits.rst

```diff
@@ -136,7 +138,7 @@ Example:
     -  It will also automatically close the issue when the commit is
        accepted into coala.
 
-.. seealso::
+.. see also::
```

Comment:
I'll remove it. i ran the file on http://rst.ninjs.org/ , i was getting error . so i thought of removing it. nevermind.